app/assets/stylesheets/govuk_publishing_components/components/_input.scss

```diff
@@ -31,19 +31,45 @@
   }
 }
 
+.gem-c-input__prefix {
```

Comment:
I would suggest reducing the repetition of these declarations by grouping them
```
.gem-c-input__prefix, .gem-c-input__suffix {
  /* common styles */
}
```
followed by
```
.gem-c-input__prefix {
  /* individual styles */
}
```

Reply:
Ahhhhhh this is what i couldn't get my head around the first time and why I used modifiers in the initial PR. Will make the change.